### ISSUE TITLE
params in expr containing op.lag raises error

### ISSUE THREAD

**Issue by bmschmidt** [uwdata/arquero#17]
While trying to figure out how to set a lag programatically, I hit some unexpected behavior.

I wanted to do:

```javascript
aq.table({"n": [1,2,3,4]})
  .params({lag: 1})
  .derive({prod: (d, $) => d.n - op.lag(d.n, $.lag)})
  .view()
```

But it raises: `TypeError: t.param is not a function`

Probably I'm just not finding the right place to set params to an op. (Or need to write a new one?).

But even the following raises an error, although the `op.lag` function does not ask for access to .params,
which seems less than ideal.

```javascript
aq.table({"n": [1,2,3,4]})
  .params({lag: 1})
  .derive({prod: (d, $) => d.n * $.lag - op.lag(d.n, 1)})
  .view()
```

**Comment by jheer** [comment]
Thanks! This definitely looks like a bug (or perhaps two!). Will investigate early next week.

**Comment by jheer** [comment]
Fixed and released in v0.13.3. Thanks again!
